disnake/ext/commands/flag_converter.py

```diff
@@ -46,6 +48,8 @@
 
     from .context import Context
 
+FlagsMetaT = TypeVar("FlagsMetaT", bound="Type[FlagsMeta]")
```

Comment:
this is no longer being used

Reply:
https://github.com/DisnakeDev/disnake/pull/1329/commits/af477853f2c74246946422a31c0b06b3e88d0ebd